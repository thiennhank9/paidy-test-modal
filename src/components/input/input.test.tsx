import TestRenderer from "react-test-renderer";
import Input from "./input";

describe("Input", () => {
  test("render Input", () => {
    const testRenderer = TestRenderer.create(<Input />);
    expect(testRenderer.toJSON()).toMatchSnapshot();
    const testInstance = testRenderer.root;
    const input = testInstance.findByType("input");
    expect(input).toBeDefined();
  });
});
