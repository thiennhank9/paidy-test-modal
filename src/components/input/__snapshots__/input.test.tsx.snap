// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input render Input 1`] = `<input />`;
